.github/workflows/release-all.yml

```diff
@@ -26,38 +26,219 @@ on:
 jobs:
   release-core:
     runs-on: ubuntu-latest
+    env:
+      WORKING_DIRECTORY: packages/core
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v4
+        with:
+          ref: develop
```

Comment:
When trying to do a manual release we encountered an error code that had to do with the inability to check out the branch on which we were currently on, develop in this case. I'm not sure exactly how this script is configured, but is that something we need to keen in mind?

Reply:
The problem in the last release was in "Manual create release PR" which was trying to use develop as a reference and reset it before bumping the version number. Now what we did in that script it so pull the latest develop and create a release branch as the first step so there should not be issues. Tim and I have tested it so the script runs OK.

In this script, when it comes to real release, we are doing everything on develop branch, there are no changes to it made, just building and publishing so I hope it will run smooth.